app/screens/user_profile/user_info.tsx

```diff
@@ -22,41 +23,27 @@ type Props = {
     enableCustomAttributes?: boolean;
 }
 
-const emptyList: DisplayCustomAttribute[] = []; /** avoid re-renders **/
+const emptyList: CustomAttribute[] = []; /** avoid re-renders **/
 
 const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user, enableCustomAttributes}: Props) => {
     const customStatus = getUserCustomStatus(user);
     const serverUrl = useServerUrl();
-    const [customAttributes, setCustomAttributes] = useState<DisplayCustomAttribute[]>(emptyList);
+    const [customAttributes, setCustomAttributes] = useState<CustomAttribute[]>(emptyList);
+
     const lastRequest = useRef(0);
 
+    const sortAttributes = useCallback(sortCustomProfileAttributes, []);
```

Comment:
`sortCustomProfileAttributes` is already a constant imported from a different file. There is no reason to use a `useCallback` here.